rllib/algorithms/ddppo/ddppo.py

```diff
@@ -51,6 +49,9 @@
 class DDPPOConfig(PPOConfig):
     """Defines a configuration class from which a DDPPO Algorithm can be built.
 
+    Note(jungong) : despite best efforts, DDPPO is not really fault tolerant and
```

Comment:
Makes sense.

Reply:
Maybe say: "It doesn't use the elasticity feature of WorkerSet."
Here, it sounds like, we don't use WorkerSet at all.

Reply:
sure, done.